doc/teacher_doc/run_file.rst

```diff
@@ -634,6 +634,22 @@ The four letter code of the student's language (for example `en_US` or `fr_FR`)
 
         getinput @lang
 
+The submission time, following the datetime format "%Y-%M-%D %H:%M:%S.%f", can be retrieved using
+
+.. tabs::
+
+    .. code-tab:: ipython3
+
+        lang = get_input("@time")
```

Comment:
Super little detail, it could be better to call the variable time in place of lang (cp/p issue probably)